pilot/pkg/networking/plugin/authz/validate.go

```diff
@@ -1,77 +0,0 @@
-// Copyright 2018 Istio Authors
```

Comment:
Why are these checking deleted?

Reply:
It's moved to `model.go`/`model_test.go`